includes/src/Report/Accordian.js

```diff
@@ -0,0 +1,326 @@
+                    }
+                }
+                count++;
+            }
+        }
+        testCaseMax = (maxUserRank > maxOWTFRank)
+            ? maxUserRank
+            : maxOWTFRank;
+        return {rank: testCaseMax, count: count};
+    };
+
+    handlePluginBtnOnAccordian(plugin_type) {
+        if (!this.state.isClicked) {
+            this.fetchData(plugin_type);
+        } else {
+            this.setState({
+                pactive: plugin_type
+            }, function() {
+                $('#' + this.state.code).collapse('show');
+            });
+        }
+    };
+
+    fetchData(plugin_type) {
+        if (this.state.isClicked === false) {
+            var target_id = document.getElementById("report").getAttribute("data-code");
+            this.serverRequest = $.get(TARGET_API_URI + target_id + '/poutput/' + '?plugin_code=' + this.state.code, function(result) {
+                this.setState({
+                    pluginData: result,
+                    isClicked: true,
+                    pactive: plugin_type
+                }, function() {
+                    $('#' + this.state.code).collapse('show');
+                });
+            }.bind(this));
+        }
+    };
+
+    patchUserRank(group, type, code, user_rank) {
+        var target_id = document.getElementById("report").getAttribute("data-code");
+        var presentState = this.state.pluginData;
+        var alert = this.alert;
+        $.ajax({
+            url: TARGET_API_URI + target_id + '/poutput/' + group + '/' + type + '/' + code,
+            type: 'PATCH',
+            data: {
+                "user_rank": user_rank
+            },
+            error: function(xhr, textStatus, serverResponse) {
+                alert.call(this, "Server replied: " + serverResponse);
+            }.bind(this)
+        });
+        var index = -1;
+        var pactive = this.state.pactive;
+        for (var i = 0; i < presentState.length; i++) {
+            if (presentState[i].plugin_group === group && presentState[i].plugin_type === type) {
+                presentState[i].user_rank = user_rank;
+                index = i;
+            }
+        }
+        if (index === presentState.length - 1) {
+            $('#' + code).collapse('hide');
+        } else {
+            pactive = presentState[index + 1]['plugin_type'];
+        }
+        this.setState({pluginData: presentState, pactive: pactive});
+    };
+
+    postToWorkList(selectedPluginData, force_overwrite) {
+        selectedPluginData["id"] = document.getElementById("report").getAttribute("data-code");
+        selectedPluginData["force_overwrite"] = force_overwrite;
+        var alert = this.alert;
+        $.ajax({
+            url: WORKLIST_API_URI,
+            type: 'POST',
+            data: $.param(selectedPluginData, true),
+            success: function(data) {
+                alert.call(this, "Selected plugins launched, please check worklist to manage :D");
```

Comment:
@delta24 alert is not here default alert of html. Here alert is a function which calls react notifier to notify like request-reponse messages etc.